### ISSUE TITLE
Bloc Grid + Responsive Gutters

### ISSUE THREAD

**Issue by andycochran** [zurb/foundation-sites#8668]
It would be rad if `margin-bottom` on columns Block Grids used Responsive Gutter sizes. Then the space between rows would match the space between columns.

**Comment by andycochran** [comment]
Something like: 

``` scss 
.[size]-up-[n] {
  .column {
    margin-bottom: [responsive gutter size];
    :last-child {
      margin-bottom: 0;
    }
  }
}
```

**Comment by rafibomb** [comment]
hmmm, this makes sense but may be a bit prescriptive. I'll shoot around to some of the designers here!

**Comment by brettsmason** [comment]
I like the idea of keeping consistent margins on the block grid as I think this would be the most common aim, though I can think of some occasions I may not want to remove the margin on the last item.

**Comment by andycochran** [comment]
I've realized that this style only makes sense for a grid that's visually a "Block Grid." But since the block grid from F5 has been merged into the main grid, there is no actual block grid. There's just shorthand for equal-width columns. And those shouldn't necessarily look like block grids. 

So I just did this w/ some app-specific styles and a custom `.block-grid` class on the `.row`: 

```scss
.block-grid {
  .columns, .column {
    margin-bottom: map-get($grid-column-gutter, small);
    @include breakpoint(medium) {
      margin-bottom: map-get($grid-column-gutter, medium);
    }
    @include breakpoint(large) {
      margin-bottom: map-get($grid-column-gutter, large);
    }
    & > :last-child {
      margin-bottom: 0;
    }
  }
}
```

I often add custom app-specific styles for `margin-bottom` on columns. Maybe my original idea could be further abstracted by a class that adds gutter-size margin to the bottom of columns?

**Comment by andycochran** [comment]
@rafibomb What does Zurb think about ^this idea (adding a class that matches the bottom margin of columns to the gutter to visually make grids "block grids")? I will PR if cool.

**Comment by kball** [comment]
I think a PR for this would be great, but make sure it is as a mixin as well as a class

**Comment by kball** [comment]
@andycochran were you going to take a look at this one?

**Comment by andycochran** [comment]
Yup. I was waiting on that fix for the responsive gutters regression. On it…

**Comment by andycochran** [comment]
@ncoden Would you advise I use the following class instead of the nested, higher-specificity classes I proposed above? 
```sass
.#{$block-grid}-#{$column} {
  // margin bottom for each breakpoint...
}
```

**Comment by ncoden** [comment]
@andycochran Yes.

.
.
.

*Thank you so much*.
